vault/request_handling.go

```diff
@@ -548,6 +548,7 @@ func (c *Core) handleCancelableRequest(ctx context.Context, ns *namespace.Namesp
 				Auth:                auth,
 				Request:             req,
 				Response:            auditResp,
+				MountType:           entry.Type,
```

Comment:
OK, just to note what we said in our offline conversation, this now seems unnecessary because we get the MountType from Request instead.